### PR TITLE
New design for blog2 header

### LINKED ISSUE #5352: Blog redesign next steps and discussion
We're looking to improve the blog template again with some design inspiration from this WordPress template:

image

We'd like to keep it really minimal but want to:

* add a sidebar including recent posts
* show only the first 300px of the post content, with a `Read more` button that toggles the rest to be visible
* include social media links in sidebar
* include a list of recent tweets in sidebar

We'd love help chopping this up into pieces to implement! This is a pretty simple set of changes that would make some good first-timers-only issues!

### PR DESCRIPTION
**Pull request by gautamig54**
Fixes #5352  (<=== Add issue number here)

Screen Shot 2019-04-21 at 6 51 57 PM

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

File: app/views/tag/blog2.html.erb [hunk 1]
```diff
@@ -13,17 +36,17 @@
     <p>
     Here is the lorem ipsum content
     <br>
-    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod 
-      tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim 
-      veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex 
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
```
Comment (CleverFool77) [inline app/views/tag/blog2.html.erb]: Hi @gautamig54 
Damn!! The UI looks great.
Regarding spaces ? We are going to remove this lorem ipsum soon.
Is it necessary ?
:thinking:

**Comment by gautamig54** [comment]
Screen Shot 2019-04-21 at 6 51 57 PM

File: app/views/tag/blog2.html.erb (hunk 1 above)
Comment (gautamig54) [reply app/views/tag/blog2.html.erb]: Thanks @CleverFool77 !!!
I think the spacing happened unintentionally. I'll resolve it.

**Comment by gautamig54** [comment]
@jywarren @gauravano Kindly review this PR. Thanks!

**Comment by gauravano** [comment]
Do you think we can improve RSS and Following by adding some space and maybe trying to position icons after that? Thanks!

**Comment by gautamig54** [comment]
@gauravano Yes I'll improve icons by adding spaces. 
Thanks for the review!

**Comment by gautamig54** [comment]
@gauravano  What do you think about this?

Screen Shot 2019-04-21 at 10 21 13 PM

**Comment by gauravano** [comment]
Hi @gautamig54, the above UI looks great, please push the changes. Thanks!

**Comment by gautamig54** [comment]
@gauravano I have pushed the changes.  Thanks!!

**Comment by jywarren** [comment]
Oh this looks great! Shall we merge this and move on to the next? Thank
you!!!

**Comment by gauravano** [comment]
Merging this but we need to make this page mobile friendly too 

Screenshot from 2019-04-30 19-17-06

**Comment by gauravano** [comment]
Merged. Thank you @gautamig54 @CleverFool77  :tada:

**Pull request merged**